catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveAllCommand.java

```diff
@@ -295,15 +267,10 @@ private QueryRequest getIntendedQuery(FilterBuilder filterBuilder, boolean isReq
     }
 
     QueryImpl query = new QueryImpl(filter);
-
     query.setRequestsTotalResultsCount(isRequestForTotal);
-
     query.setPageSize(batchSize);
 
-    Map<String, Serializable> properties = new HashMap<>();
-    properties.put("mode", "native");
```

Comment:
This mode was only ever used by the old, removed Standard UI.  I removed it since there are no plans to use this feature in the future since it was too slow and did not scale well.